test/addons/cdi/cr/kustomization.yaml

```diff
@@ -4,7 +4,7 @@
 # yamllint disable rule:line-length
 ---
 resources:
-  - https://github.com/kubevirt/containerized-data-importer/releases/download/v1.58.1/cdi-cr.yaml
+  - https://raw.githubusercontent.com/nirs/drenv-addons/main/cdi/dabbcbed1618d1ec970135b4c0a8b692c2dd60b6/cdi-cr.yaml
```

Comment:
Assume this changes once this PR is merged

Reply:
Hopefully soon